spinnaker/spinnaker_system/kube_smoke_test.py

```diff
@@ -243,22 +270,130 @@ def create_server_group(self):
     (builder.new_clause_builder('Replication Controller Added',
                                 retryable_for_secs=15)
      .get_resources('rc', extra_args=[group_name])
-     .contains_path_eq('spec/replicas', 2))
+     .contains_path_eq('spec/replicas', 1))
 
     return st.OperationContract(
         self.new_post_operation(
             title='create_server_group', data=payload, path='tasks'),
         contract=builder.build())
 
-  def delete_server_group(self):
+  def make_smoke_stage(self, requisiteStages=[], **kwargs):
+    result = {
+      'requisiteStageRefIds': requisiteStages,
+      'refId': 'FINDIMAGE',
+      'type': 'findImage',
+      'name': 'Find Valid Image',
+      'cloudProviderType': 'kubernetes', 
+      'namespaces': ['default'], 
+      'cloudProvider': 'kubernetes',
+      'selectionStrategy': 'NEWEST',
+      'onlyEnabled': True,
+      'credentials': 'my-kubernetes-account',
```

Comment:
Credentials should be obtained from the deployed configuration.  See any of the other tests for how to do this.